src/App.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { Route, Routes } from "react-router";
+
+import Header from "./components/Header/Header";
+import Home from "./components/Home/Home";
```

Comment:
I can already tell from the get go you're not messing about with every 'component should  be it's own component'

Reply:
Yeah I've not considered it till now but it is quite a few imports I've got there. 🤣